tools/pipelines-tasks/azure-pipelines/templates/build-steps.yml

```diff
@@ -20,6 +39,25 @@ steps:
   inputs:
     restoreSolution: '$(tasksRoot)/test/assets/HelloWorldUWPApp/HelloWorldApp.sln'
 
+- task: PowerShell@2
+  displayName: "Set AVD Config input data for avd publish tests" 
+  inputs:
+    targetType: 'inline'
+    script: |
+      $avdInputsConfigData = (Get-Content -Path $testAvdInputsFile) | ConvertFrom-Json
+      $avdInputsConfigData.connectedServiceNameARM = $subscriptionId
+      $avdInputsConfigData.clientId = $clientId
+      $avdInputsConfigData.clientSecret = $clientSecret
+      $avdInputsConfigData.tenantId = $tenantId
+      $avdInputsConfigData.emailId = $emailId

```

Comment:
This is the email of loggedIn ADO user so that it can be assigned to process env (process.env.BUILD_REQUESTEDFOREMAIL) which can't be derived from these id's